test/test_euler.py

```diff
@@ -750,6 +773,9 @@ def rhs(t, q):
                 write_soln()
 
         fields = rk4_step(fields, t, dt, rhs)
+        fields = filter_modally(discr, "vol", cutoff=cutoff,
+                                       mode_resp_func=frfunc, field=fields)
```

Comment:
The filter should drop the convergence order. That might be useful to test.

Reply:
Running order-2 elements on the Vortex2D case, the filter appears to improve the order of convergence.